code-challenges.js

```diff
@@ -13,16 +13,43 @@ var num3 = -7
 // Expected output: "-7 is not divisble by three"
 
 
-
+// create a function that takes in a number
+const divizByThree = (number) => {
+ // create a conditional that checks if the number modulo is = to 0
+    if (number % 3 === 0){
+        // if it is, return it is div by three
+        return `${number} is divisible by three`
+    }
+    // if number modulo is NOT = 0 
+    else {
+        // return it is not div by three
+        return `${number} is not divisible by three`
+    }
+}
+console.log (divizByThree(num1))
+console.log (divizByThree(num2))
+console.log (divizByThree(num3))
 
 
 // --------------------2) Create a function that takes in the randomNouns variable and returns an array with all the words capitalized.
 
 var randomNouns = ["streetlamp", "potato", "teeth", "conclusion", "nephew", "temperature", "database"]
 // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew", "Temperature", "Database"]
 
+// create a function that takes in an array
+const capitalizer = (array) => {
+    // create a new array to hold the capitalized words
+    let capArray = []
+    // map over the array 
+    capArray = array.map(value => {
+        //change the value at index0 of each word to a capital and then concat it to the remainder of the substring
+        return value[0].toUpperCase() + value.substring(1)
+    })
+    //return the capitalized array
+    return capArray
+}
```

Comment:
cool! this could be refactored to be a little smaller but i really like that every step is pseudocoded and i can tell exactly what you were trying to do :)